docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -507,8 +507,12 @@ Two special environment variables are passed to the render command:
 - `FORMAT`: Time format to diplay on UI. i.e. RFC1123 or 2006-01-02 15:04:05
 - `DEFAULT_UI_LOCATION`: Default location of time on the UI, so that we can display correct user's time on UI. i.e. Shanghai/Asia
 
+## Storage (`storage`)
+
+- `BUCKET_URL`: URL of Bucket where files, such as, attachments, avatars, etc. are stored. If unset, file://<current_directory> is used to maintain backward compatibility. Consult https://gocloud.dev/howto/blob/ for URL format for various cloud providers.
+
 ## Other (`other`)
 
 - `SHOW_FOOTER_BRANDING`: **false**: Show Gitea branding in the footer.
 - `SHOW_FOOTER_VERSION`: **true**: Show Gitea version information in the footer.
-- `SHOW_FOOTER_TEMPLATE_LOAD_TIME`: **true**: Show time of template execution in the footer.
\ No newline at end of file
+- `SHOW_FOOTER_TEMPLATE_LOAD_TIME`: **true**: Show time of template execution in the footer.
```

Comment:
This change does not seems to be related to this PR

Reply:
It's just a new line at the end.